api/src/main/java/org/openmrs/module/patientflags/evaluator/SQLFlagEvaluator.java

```diff
@@ -129,4 +129,73 @@ public FlagValidationResult validate(Flag flag) {
 		// if we've gotten this far, mark the criteria as valid
 		return new FlagValidationResult(true);
 	}
+
+	@Override
+	public String evalMessage(Flag flag, int patientId) {
+		String literal = "\\$\\{\\d{1,2}\\}";
+		String message = flag.getMessage();
+		System.out.println("my message ::::"+message);

```

Comment:
Let's remove the System.out.println--in areas that we really want to log a message, we should use log.info, log.warning, log.error, etc. (See FlagServiceImpl)